app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/profile/ProfileViewModel.kt

```diff
@@ -1,25 +1,24 @@
-package com.bottlerocketstudios.brarchitecture.ui.user
+package com.bottlerocketstudios.brarchitecture.ui.profile
 
 import android.content.Intent
 import androidx.core.net.toUri
-import androidx.lifecycle.asLiveData
 import com.bottlerocketstudios.brarchitecture.R
 import com.bottlerocketstudios.brarchitecture.data.converter.convertToUser
 import com.bottlerocketstudios.brarchitecture.data.repository.BitbucketRepository
 import com.bottlerocketstudios.brarchitecture.navigation.ExternalNavigationEvent
 import com.bottlerocketstudios.brarchitecture.navigation.NavigationEvent
 import com.bottlerocketstudios.brarchitecture.ui.BaseViewModel
+import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.map
 
-class UserFragmentViewModel(val repo: BitbucketRepository) : BaseViewModel() {
-
-    // TODO: Remove conversion from Flow to LiveData once it is in the next stable 7.x AGP: https://developer.android.com/topic/libraries/data-binding/observability#stateflow
-    val avatarUrl = repo.user.map { it?.convertToUser()?.avatarUrl.orEmpty() }.asLiveData()
-    val displayName = repo.user.map { it?.displayName.orEmpty() }.asLiveData()
-    val nickname = repo.user.map { it?.nickname.orEmpty() }.asLiveData()
+class ProfileViewModel(val repo: BitbucketRepository) : BaseViewModel() {
+    val avatarUrl: Flow<String> = repo.user.map { it?.convertToUser()?.avatarUrl.orEmpty() }
+    val displayName: Flow<String> = repo.user.map { it?.displayName.orEmpty() }
+    val nickname: Flow<String> = repo.user.map { it?.nickname.orEmpty() }
+    private val bitBucketSettingUrl: String = "https://bitbucket.org/account/settings/"
```

Comment:
@VanyaHuamanBottleRocket nit: Couple of things:

1. Move to a `companion object` block at the bottom of the class
2. Make it `const`
3. Use `SCREAMING_SNAKE_CASE`